src/lib/utils/APIConstants.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Separating API endpoints into their own constants file
+ */
+
+import { getEndpoint, BackendDestinationEnum } from './functions/getEndpoint'
+
+export const BACKEND_ENDPOINT = `${getEndpoint(
+  BackendDestinationEnum.REPO_ENDPOINT,
+)}`
```

Comment:
I didn't incorporate this into the strings for now, because the methods in `SynapseClient` accept the endpoint as an argument. I think it would make sense to do so eventually.